### PR TITLE
Tentative implementation for #1405

### PR DESCRIPTION
**Pull request by chiraag-nataraj**
Fix for recent versions of `xpra` not working properly.

### CODE REVIEW

**Comment by chiraag-nataraj** [comment]
I realized I never `free` `start_child`. Also, I commented out the exit code since `xpra` handles it now, but that means there isn't a good place to free `start_child`...

File: src/firejail/x11.c [hunk 1]
```diff
@@ -870,6 +862,15 @@ void x11_start_xpra(int argc, char **argv) {
 
 	/* sleep(1); // adding a delay in order to let the server start */
 
+	// wait for server to end
+	while (1) {
```
Comment (SpotComms) [inline src/firejail/x11.c]: What kind of perf impact will this have, won't it just peg a core at 100%? Can you add a `nanosleep(250)`?

**Comment by netblue30** [comment]
Let's give it a try, thanks!

**Comment by netblue30** [comment]
Some more things going in there.

We have two start_xpra() functions, the old one and the new one. I am looking around, most distos are still on Xpra v0.17. I added in /etc/firejail/firejail.config a "xpra-attach" setting. You'll have to put it to yes and it will use your version of start_xpra(). For now the setting is "no", meaning it will run the old version - we'll switch it as soon as xpra 2.x versions start appearing in distributions.

**Pull request merged**